assets/js/translator.js

```diff
@@ -133,8 +133,6 @@ if(modeEnabled('translation')) {
             synchronizeTextareaHeights();
         });
 
-        $(window).resize(synchronizeTextareaHeights);
```

Comment:
I think that commit just fixed a bunch of style issues. You'll have to look backwards in time a bit.